File: eval/fixtures/legacy-directory-backed-spring-commerce/src/main/java/com/acme/legacy/billing/BillingRefundRepository.java
package com.acme.legacy.billing;

public class BillingRefundRepository {
  public RefundAuditResult writeRefundAuditEntry(String refundId) {
    String sql = "CALL rebuild_spring_refund_audit(); SELECT * FROM billing.spring_refund_audit_entries";
    return new RefundAuditResult(sql);
  }
}

File: eval/fixtures/legacy-directory-backed-wcf-commerce/src/Legacy/Billing/BillingRefundRepository.cs
namespace Acme.Legacy.Billing;

public class BillingRefundRepository
{
  public string WriteRefundAuditEntry(string refundId)
  {
    string sql = "EXEC billing.rebuild_wcf_refund_audit; SELECT * FROM billing.wcf_refund_audit_entries";
    return sql;
  }
}
